deployments/kubernetes/infra/gateway-config.yaml

```diff
@@ -5,52 +5,42 @@ metadata:
 data:
   gateway.json: |
     {
-      "upstreams": {
-          "api": "http://api",
-          "authoring": "http://authoring:3000",
-          "publishing": "http://publishing",
-          "editor": "http://editor:3000"
-      },
-      "security": {
-          "auth": {
-              "providers": {}
-          }
-      },
-      "v1Hosts": {
-          "api": [
-              "api",
-              "api.localtest.me"
-          ],
-          "authoring": [
-              "authoring",
-              "authoring.localtest.me"
-          ]
-      }
+        "security": {
+            "auth": {
```

Comment:
Do we need security section? or in general this gateway.json? maybe have just one